parsons/databases/redshift/redshift.py

```diff
@@ -401,9 +401,12 @@ def copy(self, tbl, table_name, if_exists='fail', max_errors=0, distkey=None,
                 characters.
             statupate: boolean
                 Governs automatic computation and refresh of optimizer statistics at the end
-                of a successful COPY command.
+                of a successful COPY command. If ``True`` explicitly sets ``statupate`` to on, if
+                ``False`` explicitly sets ``statupate`` to off. Defaults to ``None``.
             compupdate: boolean
-                Controls whether compression encodings are automatically applied during a COPY.
+                Controls whether compression encodings are automatically applied during a COPY. If
+                ``True`` explicitly sets ``compupdate`` to on, if ``False`` explicitly sets
+                ``compupdate`` to off. Defaults to ``None``.
```

Comment:
Perhaps explain to folks what happens if it is set to `None`. Or at least link to the redshift documentation?